app/src/main/java/com/realityexpander/tasky/core/common/settings/AppSettings.kt

```diff
@@ -0,0 +1,24 @@
+package com.realityexpander.tasky.core.common.settings
```

Comment:
`core.common` seems duplicated since `core` and `common` usually refer to the same thing. Settings are data-related and should therefore go in `core->data`

Reply:
OK I will remove the extraneous `common` package, and move the settings.